packages/components/organisms/f-takeawaypay-activation/src/components/TakeawaypayActivation.vue

```diff
@@ -1,28 +1,65 @@
 <template>
-    <div
-        :class="$style['c-takeawaypayActivation']"
-        data-test-id="takeawaypayActivation">
-        {{ copy.text }}
+    <div :class="$style['c-takeawaypayActivation']">
+        <card-component
+            is-rounded
+            is-page-content-wrapper
+            card-heading-position="center"
+            data-test-id="takeawaypay-activation-component"
+            :class="$style['c-takeawaypayActivation-card']">
+            <activation-logged-in
```

Comment:
This seems like you could be able to use `component is=""` to make this a single dynamic component rather than initialising all of these components? (it looks like this would only show one of these views at once but can't see from a quick look how you show/hide the different states?)

Reply:
@ashleynolan Yep, only 1 component will be displayed based on the HTTP call during initialization. This logic will be added in the next task and here was just some preparation